modules/sov-ibc-transfer/src/context.rs

```diff
@@ -269,16 +317,25 @@ where
         channel_id: &ChannelId,
         coin: &PrefixedCoin,
     ) -> Result<(), TokenTransferError> {
+        let token_id = TokenId::from_str(&coin.denom.to_string()).map_err(|e| {
+            TokenTransferError::InvalidCoin {
+                coin: coin.to_string(),
+            }
+        })?;
+
+        if let Some(token_name) = self
+            .ibc_transfer
+            .minted_token_id_to_name
+            .get(&token_id, *self.working_set.borrow_mut())
+        {
+            return Err(TokenTransferError::Other(format!(
+                "Token with ID '{token_id}' is an IBC-created token and cannot be unescrowed.\
+                    Use '{token_name}' as denom for receiving an IBC token from the source chain"
```

Comment:
```suggestion
                Use '{token_name}' as denom for receiving an IBC token from the source chain"
```

just being pedantic about aligning.

Reply:
[e9651e1](https://github.com/informalsystems/sovereign-ibc/pull/133/commits/e9651e12c50544acd44a1d4ce4455c65911aebba)